pyaedt/modeler/Object3d.py

```diff
@@ -498,23 +506,28 @@ def normal(self):
 
 
 class Object3d(object):
-    """Object Attributes Manager for the AEDT 3D Modeler
+    """Manages object attributes for the AEDT 3D Modeler.
+    
+    Parameters
+    ----------
+    parent :
+    name :
 
+    Examples
+    --------
     Basic usage demonstrated with an HFSS design:
 
-    ##################################################
     >>> from pyaedt import Hfss
     >>> aedtapp = Hfss()
     >>> prim = aedtapp.modeler.primitives
 
+    Create a part, such as box, to return an :class:`pyaedt.modeler.Object3d.Object3d`.
 
-    Then we can create a part. Create_box returns an Object3d object
-
-    ##################################################
     >>> id = prim.create_box([0, 0, 0], [10, 10, 5], "Mybox", "Copper")
     >>> part = prim[id]
 
-    Properties
+    Properties are:
```

Comment:
This line and the following list can be removed.  We automatically generate a list within our sphinx docgen.